_maps/map_files/rockhill/rockhill.dmm

```diff
@@ -6261,16 +6293,12 @@
 /turf/open/water/cleanshallow,
 /area/rogue/under/town/sewer)
 "gMQ" = (
-/obj/structure/closet/crate/roguecloset,
-/obj/item/natural/feather,
-/obj/item/paper,
-/obj/item/paper,
-/obj/item/paper,
-/obj/item/paper,
-/obj/item/paper,
-/obj/item/paper,
-/obj/item/paper,
-/obj/item/paper,
+/obj/structure/closet/crate/chest{
+	lockid = "keep_armory"
```

Comment:
same as above

Reply:
whats your discord, ill ping you when its fixed

Reply:
resolved but needs re-review